charts/metering-ansible-operator/templates/olm/catalogsourceconfig.yaml

```diff
@@ -1,11 +0,0 @@
-apiVersion: operators.coreos.com/v1
```

Comment:
We don't want this file deleted, but adapted to what you did in the previous PR and renamed to the `catalogsource.yaml`.

Reply:
@timflannagan1 so you want both templates to be there i.e. catalogsourceconfig.yaml and catalogsource.yaml?

Reply:
Hmm my diff only showed one file when I was looking at it.

Reply:
yeah, renamed the file in the template, if we are not using `catalogsourceconfig` then I think we should remove the old `catalogsourceconfig.yaml` that was there  before this PR